src/scripts/main.js

```diff
@@ -1,3 +1,79 @@
 'use strict';
 
 // write code here
+
+const table = document.getElementsByTagName('table');
+
+const people = table[0].children[1].children;
+
+const headers = [...table[0].children[0].children[0].children];
+
+table[0].children[0].children[0].addEventListener('click', () =>
+  sortColumn(headers.indexOf(event.target))
+);
+
+function getClearData(data, index) {
+  let newIndex = index;
+
+  if (index === 1) {
+    newIndex = 0;
+  }
+
+  let clearData;
+
+  switch (newIndex) {
+    case 0:
```

Comment:
```suggestion
    case 0:
    case 1:
```

instead of this code:
```
if (index === 1) {
    newIndex = 0;
  }
```